### DISCUSSION TITLE
Change the start time?

### DISCUSSION THREAD

**Discussion by cxantia** [gnmyt/MySpeed#883] [General]
Hello,

thanks for your work in MySpeed. I like it a lot. I have a question. I have two MySpeed ​​instances running - one on the internet router and one behind a Powerlan / WLAN bridge. Both instances measure the speed every 3 hours. The results influence each other. Where can I influence the start time so that the instances run asynchronously? I looked for a config file but couldn't find anything.

Thank you in advance for an answer!
Helmer

**Comment by daubman** [comment]
I would like this feature as well - even measuring on-the-hour is likely to be a contention time against the public servers, being able to set an offset (or, even better, random "jitter" around the timing like +/- 7 minutes hourly) would be very useful.
